code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -42,3 +38,40 @@
 
 	if(istype(A, /obj/structure/headpike))
 		owner.current.add_mood_event("oogabooga", /datum/mood_event/sacrifice_good)
+
+/datum/team/ashwalkers
+	name = "Ash Walker Tribe"
+	member_name = "Ash Walker"
+	///A list of "worthy" (meat-bearing) sacrifices made to the Necropolis
+	var/sacrifices_made = 0
+	///A list of how many eggs were created by the Necropolis
+	var/eggs_created = 0
+
+/datum/team/ashwalkers/roundend_report()
+	var/list/report = list()
+
+	report += "<span class='header'>An Ash Walker Tribe inhabited the wastes...</span><br>"
+	if(length(members)) //The team is generated alongside the tendril, and it's entirely possible that nobody takes the role.
+		report += "The [member_name]s were:"
+		report += printplayerlist(members)
+
+		var/datum/objective/protect_object/necropolis_objective = locate(/datum/objective/protect_object) in objectives
+
+		if(necropolis_objective)
+			objectives -= necropolis_objective //So we don't count it in the check for other objectives.
+			report += "<b>The [name] was tasked with defending the Necropolis:</b>"
```

Comment:
these could reasonably be `span_bold()`, pretty sure our HTML ships with class=bold (if it doesn't: wack).

Reply:
I tried using span_bold(), it didn't seem to do anything.

I concur, this is indeed "wack".